src/ui/globals/types.ts

```diff
@@ -69,6 +69,7 @@ enum ToastMsgType {
   NEW_CONNECTION_ADDED = "newConnectionAdded",
   CREDENTIAL_REQUEST_PENDING = "credentialRequestPending",
   NEW_CREDENTIAL_ADDED = "newCredentialAdded",
+  CREDENTIAL_REVOKED = "credentialRevoked",
```

Comment:
hmm the acceptance criteria was to create a new notification, not a new toast message

https://www.figma.com/design/XN6K76yt13zxCUHPh1Egr8/Identity-wallet?node-id=12468-55377&t=OBZyUkuZ7x69BCJU-0

Reply:
is there a toast message somewhere in the designs that i've missed?

Reply:
Oh, sorry, I misunderstand that criteria.

Reply:
hmmm, so we will consider this notification as read already, right?